_middle/supervising.md

```diff
@@ -1,12 +1,49 @@
 ---
 layout: post
-title: "Supervising semester projects"
+title: "Supervising projects"
 category: middle
 order: 3
 ---
 
-Bachelor and Master students at EPFL must complete “semester projects” in labs. They are worth a lot of credits, so they spend significant time on them.
+## Research projects
 
-PhD students are often tasked with coming up with project topics, and more importantly supervise the students. This typically means meeting with them once per week to check their progress, help them get unstuck, etc.
+Master's students at EPFL must complete “research projects” in labs, as well as some Bachelor's students.
```

Comment:
AFAIK it is now entirely optional for Bachelor students with the new curriculum reform? But not 100% sure about this.

Reply:
It's optional for IN students but still mandatory for SC.

Reply:
LGTM